test/e2e/framework/log_test.go

```diff
@@ -117,16 +117,16 @@ func TestFailureOutput(t *testing.T) {
 		},
 	}
 	// Compare individual fields. Comparing the slices leads to unreadable error output when there is any mismatch.
-	g.Expect(len(actual)).To(gomega.Equal(len(expected)), "%d entries in %v", len(expected), actual)
```

Comment:
need to remove line 86 due to `b/f/w/test/e2e/framework/log_test.go:86:2: g declared and not used`

Reply:
Thanks, already updated.